controlplane/kubeadm/internal/etcd/util/util.go

```diff
@@ -0,0 +1,45 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import (
+	"sigs.k8s.io/cluster-api/controlplane/kubeadm/internal/etcd"
+)
+
+// MemberStarted checks whether a member has started.
+func MemberStarted(member *etcd.Member) bool {
+	return len(member.Name) > 0
```

Comment:
Is this enough to understand if an etcd member is actually running? In which case a member wouldn't have a name?

Reply:
This is unused and will be removed